src/main/java/com/GeorgesServer/app/RequestHandler.java

```diff
@@ -4,9 +4,17 @@
 
 public class RequestHandler {
     public String handle(HashMap<String, String> clientRequest) {
+
+        String defaultResponse = clientRequest.get("HttpVersion") + " 200 OK" + "\n";
+
+        if (clientRequest.get("Method").equals("POST") && clientRequest.get("Url").equals("/form")) {
```

Comment:
Kind of a smell to have two identical returns for different conditionals, but you're pretty early in this process so :shrug: Clean this up as it grows.